addons/mail/static/src/models/call_participant_card.js

```diff
@@ -187,24 +198,17 @@ registerModel({
             compute: '_computeCallView',
             inverse: 'participantCards',
         }),
-        /**
-         * The call view for which this card is the main card.
-         */
-        callViewAsMainCard: one('CallView', {
-            inverse: 'mainParticipantCard',
-            readonly: true,
+        rtcSession: one('RtcSession', {
+            related: 'channelMember.rtcSession',
+            inverse: 'callParticipantCards',
         }),
         /**
-         * The call view for which this card is one of the tiles.
+         * The call view for which this card is the main card.
```

Comment:
comment looks wrong: main card for side bar field? also "the main card" as if there is only one?